### DISCUSSION TITLE
Set cursor position

### DISCUSSION THREAD

**Discussion by raffaelemancuso** [hyprwm/Hyprland#1257] [Q&A] [answered]
How can I set cursor position with hyprctl?

**Comment by vaxerski** [comment, marked as answer]
you cant

**Reply by raffaelemancuso** [reply]
can we have this feature?

**Reply by vaxerski** [reply]
why?

**Reply by raffaelemancuso** [reply]
Because I want to hide my cursor when i'm reading

**Reply by vaxerski** [reply]
cursor timeout exists

**Reply by raffaelemancuso** [reply]
can you give me a link?

**Reply by vaxerski** [reply]
https://wiki.hyprland.org/Configuring/Variables/#general

**Reply by raffaelemancuso** [reply]
Thanks!

**Reply by latipun7** [reply]
Sorry to necrobump this.

When I first start Hyprland, the cursor default at the top left corner of monitor. I'd like to see the first cursor at the center. How to do that, @vaxerski ? Any option to do that?

**Reply by vaxerski** [reply]
no

**Comment by glxguard** [comment]
My idea is to spawn the floating window in some position and 0.1s before this moment set hyprland statement focus_on_activate to true, and then turn it off

The programmer need to think like an inventor and engineer. You have some commands-make other with their help (:

**Comment by simonkovtyk** [comment]
Found this today, but wasnt helpful.

I found this solution: `hyprctl dispatch movecursor [x] [y]`
